plugins/rest/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.18...3.23 FATAL_ERROR)
+
+project(
+  rest
+  VERSION 1.0.0
+  DESCRIPTION "REST plugin for VAST"
+  LANGUAGES CXX)
+
+include(CTest)
+
+file(GLOB_RECURSE rest_sources CONFIGURE_DEPENDS
+     "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp"
+     "${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp")
+
+file(GLOB_RECURSE rest_tests CONFIGURE_DEPENDS
+     "${CMAKE_CURRENT_SOURCE_DIR}/tests/*.cpp")
+
+find_package(VAST REQUIRED)
+VASTRegisterPlugin(
+  TARGET rest
+  ENTRYPOINT src/plugin.cpp
+  SOURCES ${rest_sources}
+  TEST_SOURCES ${rest_tests}
+  INCLUDE_DIRECTORIES include)
+
+find_package(fmt REQUIRED)
```

Comment:
@dominiklohmann I'm not sure why this line is required

Reply:
I cannot reproduce this locally, but looking at the snippet where we link against it there are some other cleanups we can do:

https://github.com/tenzir/vast/blob/e42a9652e50833c5ba53bbd34d6ff12892a1cfa4/libvast/CMakeLists.txt#L432-L443

We actually require {fmt} >= 7 afaict, so maybe it's got something to do with that?